### PR TITLE
Fixes #69 documentation for colin

### LINKED ISSUE #69: Polish documentation
Update documentation before we can ship it:
Ideas:
 - Provide lists of checks for ecosystem
 - Show how it can be used as python module

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request adds a documentation for colin.
It contains these pages:
* index.rst ... index page with usage, content and about section.
* check_list.rst ... page with list of checks for fedora and redhat.
* python_api.rst ... TBD

### CODE REVIEW

File: docs/list_of_checks.rst [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+       -> optional
+
+    maintainer_label_required
+       -> Label 'maintainer' has to be specified.
+       -> The name and email of the maintainer (usually the submitter).
+       -> https://fedoraproject.org/wiki/Container:Guidelines#LABELS
+       -> ['maintainer', 'label', 'required']
+       -> optional
+
+    DOCKERFILE:
```
Comment (lachmanfrantisek) [inline docs/list_of_checks.rst]: Since we are not checking dockerfiles, this is a little bit confusing

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Colin is a tool to check generic rules/best-practices for containers/images/dockerfiles.
+
+CLI Usage
+---------
+
+This is how you can use colin afterwards:
+
+.. code-block:: bash
+
+    $ colin -h
```
Comment (lachmanfrantisek) [inline docs/index.rst]: This output is outdated. Look at the actual README.

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      -s, --stat                    Print statistics instead of full results.
+      -v, --verbose                 Verbose mode.
+      -h, --help                    Show this message and exit.
+
+
+Let's give it a shot:
+
+.. code-block:: bash
+
+    $ colin -f ./config/redhat.json rhel7
```
Comment (lachmanfrantisek) [inline docs/index.rst]: s/config/rulesets

File: docs/list_of_checks.rst (hunk 1 above)
Comment (lachmanfrantisek) [reply docs/list_of_checks.rst]: We can probably remove this part for now.

File: docs/list_of_checks.rst [hunk 4]
```diff
@@ -0,0 +1,431 @@
+List of all checks
+==================
+
+Colin checks several labels in Fedora and Red Hat images.
```
Comment (lachmanfrantisek) [inline docs/list_of_checks.rst]: There are printed also non-label checks (e.g. helpfile).

Comment (lachmanfrantisek) [reply docs/list_of_checks.rst]: (The best practises part.)

File: docs/list_of_checks.rst (hunk 1 above)
Comment (phracek) [reply docs/list_of_checks.rst]: The list was generated by colin ;)

Comment (lachmanfrantisek) [reply docs/list_of_checks.rst]: Yes, we should implement it or delete the checks to not see that...;-)

File: docs/list_of_checks.rst [hunk 5]
```diff
@@ -1,7 +1,7 @@
 List of all checks
 ==================
 
-Colin checks several labels in Fedora and Red Hat images.
+Colin checks several labels, non-labeled checks (e.g.helpfile) and the best practises in Fedora and Red Hat images.
```
Comment (lachmanfrantisek) [inline docs/list_of_checks.rst]: We place helpfile check to best practices, but it's not so important.

**Pull request merged**